### PR TITLE
 schedule: actively push operator (#1536)

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve? 
cherry pick actively push operator

=

### CODE REVIEW

File: server/operator_queue.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"github.com/pingcap/pd/server/schedule"
+	"time"
```
Comment (disksing) [inline server/operator_queue.go]: not well ordered?

**Comment by rleungx** [comment]
/run-all-tests tidb=release-2.1 tikv=release-2.1 tidb-test=release-2.1

**Pull request merged**